packages/components/src/styles/checkbox-radio-styles.ts

```diff
@@ -117,21 +117,24 @@ export const getCheckboxRadioJssStyle = (
           ...(!isHighContrastMode &&
             hoverMediaQuery({
               '&(input:hover), .text:hover ~ &(input)': {
-                transition: 'unset', // Fixes chrome bug where border-color is stuck on hover color
                 borderColor: uncheckedHoverColor,
                 ...prefersColorSchemeDarkMediaQuery(theme, {
                   borderColor: uncheckedHoverColorDark,
                 }),
               },
               '&(input:checked:hover), .text:hover ~ &(input:checked)': {
-                transition: 'unset', // Fixes chrome bug where border-color is stuck on hover color
                 borderColor: checkedHoverColor,
                 backgroundColor: checkedHoverColor,
                 ...prefersColorSchemeDarkMediaQuery(theme, {
                   borderColor: checkedHoverColorDark,
                   backgroundColor: checkedHoverColorDark,
                 }),
               },
+              '.text:hover ~ &(input)': {
+                ...supportsChromiumMediaQuery({
+                  transition: 'unset', // Fixes chrome bug where border-color is stuck on hover color
```

Comment:
Wouldn't resetting just the `border-color` transition more precise?
And I guess this selector is now strong enough to handle both cases, `input` and `input:checked`?

Reply:
All transition properties can get stuck, i updated the comment. Selector is strong enough for both.